### PR TITLE
feat: Prep for SDK v3

### PR DESCRIPTION
**Pull request by AbhiPrasad**
Release for https://github.com/getsentry/sentry-fullstory/pull/92

- Use bunchee instead of rollup and change export conditions
- Re-organize package.json
- Switch LICENSE to MIT
- Write migration guide for v2 -> v3
- Update README examples
- Add CHANGELOG entry
- Update deps for TypeScript and @sentry/types

resolves https://github.com/getsentry/sentry-fullstory/issues/88
resolves https://github.com/getsentry/sentry-fullstory/issues/91

### CODE REVIEW

**Comment by AbhiPrasad** [comment]
@seanparmelee let me know what you think!

File: README.md [hunk 1]
```diff
@@ -35,30 +37,27 @@ import * as Sentry from '@sentry/browser';
 // import * as Sentry from '@sentry/react-native';
 import * as FullStory from '@fullstory/browser';
```
Comment (seanparmelee) [inline README.md:38]: not sure if you want to address this as part of the v3 upgrade, but this changed with `@fullstory/browser` v2. related PR: https://github.com/getsentry/sentry-fullstory/pull/90

Comment (AbhiPrasad) [reply README.md:38]: Yes let me get that in as well!

**Review by seanparmelee — APPROVED** [review]
Not that it counts for anything, but here's another approval 😄

**Review by Lms24 — APPROVED** [review]

**Comment by AbhiPrasad** [comment]
triggered a release - https://github.com/getsentry/publish/issues/3910

whenever that gets approved by admins should be published to npm!

**Comment by AbhiPrasad** [comment]
Released! https://www.npmjs.com/package/@sentry/fullstory/v/3.0.0

**Comment by seanparmelee** [comment]
Thanks @AbhiPrasad!!

**Pull request merged**